cmake/modules/RootNewMacros.cmake

```diff
@@ -495,7 +495,7 @@ function (ROOT_CXXMODULES_APPEND_TO_MODULEMAP library library_headers)
 
   set(modulemap_entry "module \"${library}\" {")
   # For modules GCocoa and GQuartz we need objc context.
-  if (${library} MATCHES "(libGCocoa|libGQuartz)\..*")
+  if (${library} MATCHES "(libGCocoa|libGQuartz)\\..*")
```

Comment:
Could you propagate this change to all other occurrences of `(libGCocoa|libGQuartz)`?